<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JointSettingsDialog</class>
 <widget class="QDialog" name="JointSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="fk_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>650</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Forward Kinematics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>440</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="reset_btn">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Finger Joints :</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Joint 5 :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider2">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider8">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Joint 4 :</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Joint 11 :</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Joint 8 :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider1">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider5">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Joint 3 :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Joint 6 :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider3">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Joint 2 :</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Joint 9 :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Joint 1 :</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Joint 7 :</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Joint 10 :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider6">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider11">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider9">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider4">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider10">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QSlider" name="finger_slider7">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Arm Joints :</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Joint 1 :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSlider" name="slider1">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Joint 2 :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Joint 3 :</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Joint 4 :</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Joint 5 :</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Joint 6 :</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Joint 7 :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QSlider" name="slider2">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSlider" name="slider3">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSlider" name="slider4">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QSlider" name="slider5">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QSlider" name="slider6">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QSlider" name="slider7">
            <property name="minimum">
             <number>-90</number>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>ARM :</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QSlider" name="arm_select_slider">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-color: rgb(138, 226, 52);
color: rgb(164, 0, 0);</string>
            </property>
            <property name="maximum">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>LEFT</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>RIGHT</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>groupBox_3</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox_2</zorder>
      <zorder>reset_btn</zorder>
      <zorder>horizontalSpacer</zorder>
     </widget>
     <widget class="QWidget" name="ik_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>650</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Inverse Kinematics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Left Arm</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="1">
             <widget class="QSlider" name="y_ik_left_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>x : </string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="z_ik_left_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>z :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="x_ik_left_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>y :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QDial" name="rx_ik_left_dial"/>
            </item>
            <item row="0" column="1">
             <widget class="QDial" name="ry_ik_left_dial"/>
            </item>
            <item row="0" column="2">
             <widget class="QDial" name="rz_ik_left_dial"/>
            </item>
            <item row="1" column="0" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_24">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot x :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_25">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot y :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_26">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot z :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Right Arm</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>x :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="x_ik_right_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>y :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="y_ik_right_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>z :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="z_ik_right_slider">
              <property name="minimum">
               <number>-500</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="1" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_30">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot x :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDial" name="rx_ik_right_dial"/>
            </item>
            <item row="0" column="2">
             <widget class="QDial" name="ry_ik_right_dial"/>
            </item>
            <item row="1" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_31">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot y :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_32">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rot z :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QDial" name="rz_ik_right_dial"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
